Issue 4678 - Add missing style helpers unit tests

myroslavve opened the pull request:
# Description

Added all missing unit tests for style helpers.
For `getIconStyles` did a little refactoring.
<!---
Please include a summary of the changes and the related issue.
Please also include relevant motivation and context.
--->

Fixes #4678 
# How Has This Been Tested?
Checked that all unit tests pass.
<!---
Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration.
--->

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Introduced new unit tests for various styling functions, including `getButtonIconStyles`, `getCustomCssObject`, `getFlexStyles`, `getIconPathStyles`, `getIconStyles`, `getImgWidthStyles`, `getPaginationStyles`, and `getShapeStyles`.
	- Added functionality to generate random indices and improved entity retrieval logic.

- **Bug Fixes**
	- Enhanced error handling and clarity in the `getDCEntity` function.

- **Documentation**
	- Added JSDoc comments for the `getIconStyles` function to clarify parameters and return types.

- **Style**
	- Improved readability in the `getIconPathStyles` function by reformatting code.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

myroslavve committed:
Add missing style helpers unit tests

myroslavve committed:
Change mock paths to absolute

elzadj reviewed (changes requested):
https://github.com/maxi-blocks/maxi-blocks/actions/runs/12436124185/job/34723337191

Getting this on master too, but maybe it's a good time to fix it? 

```
PASS src/extensions/text/formats/test/flatFormatsWithClass.js (7.084 s)
  ● Console

    console.error
      Error loading integration options: TypeError: window.fetch is not a function
```

myroslavve committed:
Mock `DC/constants.js` to remove errors from unit tests

There is code that is executed on top level in `constants.js` which was throwing errors
